qemux86-64.robot

```diff
@@ -11,7 +11,7 @@ Preparing test environment
   Set Suite Variable    ${RESINRC_RESIN_URL}    %{RESINRC_RESIN_URL}
   Set Suite Variable    ${image}    %{image}
   File Should Exist     ${image}  msg="Provided images file does not exist"
-  Set Suite Variable    ${application_repo}    https://github.com/resin-io-projects/alpine-barebone.git
+  Set Suite Variable    ${application_repo}    https://github.com/villaseca/kernel-server.git
```

Comment:
Yes, that repo is provisional and for prototyping. We need a helper repo that contains all the application side of the testings. For starters, accepting commands through HTTP requests and executing them.